tooltester/src/main/java/io/dockstore/tooltester/jenkins/JenkinsPipeline.java

```diff
@@ -0,0 +1,126 @@
+package io.dockstore.tooltester.jenkins;
+
+/**
+ * @author gluu
+ * @since 08/02/17
+ */
+
+import java.util.List;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class JenkinsPipeline {
```

Comment:
Ok, I would would still put the annotation on, but I think you can comment in Javadoc that it wasn't a particularly clean procedure. And in markdown, explain how a future developer might re-do it if there is a new release of the BlueOcean plugin for example.